portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,11 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
```

Comment:
Consider updating the variable name, now that it doesn't contain any greetings.

Reply:
Got it I'll update the variable names and comments in a new pull request.